<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoAudioDialog</class>
 <widget class="QDialog" name="VideoAudioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_poweroff">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>挂断</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/tupian/audio.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_minImg">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_audio">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>麦克风</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_video">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>摄像头</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
